mne/utils/docs.py

```diff
@@ -376,6 +376,42 @@
     Frequency-domain window to use in resampling.
     See :func:`scipy.signal.resample`.
 """
+docdict['decim'] = """
+decim : int
+    Factor by which to subsample the data.
+
+    .. warning:: Low-pass filtering is not performed, this simply selects
+                 every Nᵗʰ sample, i.e., compresses the signal (see Notes).
```

Comment:
Is `Nᵗʰ` intentional? I generally like Unicode, but here it might not be necessary. Also, it is not clear what `N` refers to (or maybe it is clear that it refers to `decim`, but then why not be explicit about it).

Reply:
Yes, the old docs said "this simply selects every nth sample" so this is just meant to be a minor improvement to that wording. I guess we could say `"simply subselects one sample for every `decim` samples" or something, but I'm not sure it's clearer. What wording would you find clearer?

Reply:
Hm. Nth sample is good, maybe we should clarify that N corresponds to the decim argument. Also, maybe we should write Nth and not Nᵗʰ (looks a bit weird with certain fonts)?